packages/mapper/src/mapper.ts

```diff
@@ -140,32 +141,22 @@ export default class Mapper {
     const db = collection._database._name;
     const coll = collection._name;
 
-    const dbOptions: any = {};
+    const dbOptions: DatabaseOptions = {};
 
     if ('readConcern' in options) {
       dbOptions.readConcern = options.readConcern;
     }
     if ('writeConcern' in options) {
-      dbOptions.writeConcern = options.writeConcern;
```

Comment:
the `writeConcern` option gets filtered out:

https://github.com/mongodb/node-mongodb-native/blob/fa052a514be0a485831fe5e01c3c6eafe2b23140/lib/db.js#L71

we merge `w, j, ...` directly